dockerfiles/str/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get install -qqy \
+    python3 \
+    python3-pip \
+    python \
+    python-dev \
+    awscli \
+    build-essential \
+    git \
+    libbz2-dev \
+    liblzma-dev \
+    make \
+    pkg-config \
+    wget \
+    unzip \
+    zlib1g-dev
+
+RUN pip3 install pybedtools pyvcf scipy numpy
```

Comment:
Ideally I would peg all dependencies to specific versions, if possible. This can help us rule out changes to dependencies as a reason for differences in output with future builds.

Reply:
That makes sense; I pinned `pybedtools`, `pyvcf`, `scipy`, and `numpy` to their current latest releases.